modules/hdf5_handler/h5get.cc

```diff
@@ -2546,3 +2552,14 @@ string handle_string_special_characters_in_path(const string &instr) {
     return outstr;
 }
 
+string invalid_type_error_msg(
+        string var_type
+){
+    stringstream msg;
+
+    msg << "Your request was for a (DAP2 data model response). ";
```

Comment:
Let's change this -> "Your request was for a (DAP2 data model response). "
To  "Your request was for a response that uses the DAP2 data mode. "

Reply:
Good suggestion.  Made it '... response that uses the DAP2 data model.'